.github/workflows/tests.yaml

```diff
@@ -386,7 +386,6 @@ jobs:
                      --ota-requestor-app ./out/darwin-x64-ota-requestor-${BUILD_VARIANT}/chip-ota-requestor-app \
                      --tv-app ./out/darwin-x64-tv-app-${BUILD_VARIANT}/chip-tv-app \
                      --bridge-app ./out/darwin-x64-bridge-${BUILD_VARIANT}/chip-bridge-app \
-                     --lit-icd-app ./out/darwin-x64-lit-icd-${BUILD_VARIANT}/lit-icd-app \
```

Comment:
Not sure i understand the build removal?

Reply:
test_definition.py applies in both darwin and linux chip-tool, I have not yet figured out way to check which tool is used in test_definition.py via os. We have not yet add LIT support in darwin chip-tool so that it complains icd-registration option cannot be found, we should add this back when we have lit support in darwin chip-tool.
cc @bzbarsky-apple for darwin support

Reply:
I don't follow.  We have three possible clients here:

1. darwin-framework-tool
2. chip-tool on Linux
3. chip-tool on Darwin 

Item 1 does not support the LIT ICD bits, and has TestIcdManagementCluster turned off when running with it as client.

Items 2 and 3 support the ICD stuff fine, no?  It's the same code.

If you need to tell apart 1 and 2/3 in test_definition.py, check for `test_runtime == TestRunTime.DARWIN_FRAMEWORK_TOOL_PYTHON` (or !=, depending on what you want).

But you should not need any changes to `.github/workflows/tests.yaml` here.

Reply:
yes, the below is what i have seen, I wanna disable the test for darwinframeworktool

'/Users/runner/work/connectedhomeip/connectedhomeip/scripts/tests/yaml/darwinframeworktool.py', 'pairing', 'code', '0x12344321', 'MT:-24J0AFN00KA0648G00', '--icd-registration', 'true', '--server_path', './out/darwin-x64-darwin-framework-tool-no-ble/darwin-framework-tool', '--server_arguments', 'interactive server --storage-directory /var/folders/bc/7ycrpl196j7dsckgtn2m6g800000gn/T/tmp5g7y6x7b']

Reply:
updated, thanks